datumaro/components/media.py

```diff
@@ -82,26 +126,26 @@ def __init__(self, path: str, *args, **kwargs):
     @property
     def path(self) -> str:
         """Path to the media file"""
-        return self._path
+        # TODO: do we need this replace?
+        return self._path.replace("\\", "/")
```

Comment:
Is this okay for both windows and linux?

Reply:
the backslash is an allowed character for filename in Linux so if it is the case, it would be a big problem.

Replacing backslash with slash is spread out the entire codebase. I believe it would be better to look into it further in another PR.